src/tests/work-item/work-item-list/page-objects/common.page.js

```diff
@@ -153,6 +153,12 @@ class CommonPage {
     this.microservicesMenuTab.click(); 
     return new MicroservicesPage();
   }
+  clickMobileViewHeaderMenu () {
+    return element(by.id('header-toggle-menu-btn')).click();
```

Comment:
I am able to find the button in the page html, but I am not able to see the button on the screen. Is it not displayed intentionally?